bundle/config/mutator/python/python_mutator.go

```diff
@@ -266,7 +290,23 @@ func writeInputFile(inputPath string, input dyn.Value) error {
 	return os.WriteFile(inputPath, rootConfigJson, 0600)
 }
 
-func loadOutputFile(rootPath string, outputPath string) (dyn.Value, diag.Diagnostics) {
+// loadLocationsFile loads locations.json containing source locations for generated YAML.
+func loadLocationsFile(locationsPath string) (*pythonLocations, error) {
+	if _, err := os.Stat(locationsPath); os.IsNotExist(err) {
+		return newPythonLocations(), nil
+	}
```

Comment:
You can check for this error as part of `os.Open`. Saves a syscall.

Also make sure to use `errors.Is(err, fs.ErrNotExist)` -- `os.IsNotExists` is marked deprecated.